pkg/kernels/kernels_common.go

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
whats the difference between kernels_common.go and kernels.go? I though the pattern is:

kernels.go -> common code
kernels_linux.go -> linux code
kernels_windows.go -> windows code

Reply:
kernels.go -> Linux and Mac ( Non, windows, as indicated by //go:build directive added in this [commit](https://github.com/cilium/tetragon/commit/218b6fae9afb511d921751bb5f1d6dc82e2c2af9)
kernels_common.go -> All three OS 
kernels_linux.go -> Linux
kernels_windows.go -> Windows.

Reply:
Just curious, why do you need darwin builds, does Tetragon actually builds on darwin? Maybe having kernels.go and `_linux` and `_windows` would be simpler than those combination.

Reply:
I just learned that you can use the `//go:build unix` (but not the `_unix.go` suffix) if that can help

Reply:
Yes `cmd/tetra` does build on Darwin. There is CI for it. it imports a bunch of these packages.